libcodechecker/analyze/analyzers/config_handler_clang_tidy.py

```diff
@@ -105,3 +105,25 @@ def get_checker_configs(self):
             LOG.debug(ex)
 
         return res
+
+    def enable_checker(self, checker_name, description=None):
+        """
+        Enable checker, keep description if already set.
+        """
+        if checker_name.startswith("W"):
+            self.add_checker(checker_name, True, None)
+            return
+
+        super(ClangTidyConfigHandler, self).enable_checker(checker_name,
+                                                           description)
+
+    def disable_checker(self, checker_name, description=None):
+        """
+        Disable checker, keep description if already set.
+        """
+        if checker_name.startswith("Wno-"):
```

Comment:
We should check if the checker name starts with `W` and is disabled and disable that warning too:
`--disable Wunused-parameter` or `--disable Wno-unused-parameter`